index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+  <header class="nav">
+    <a href="">Henschel</a>
+    <nav><a href=""><img src="images/more.png" alt="More" width="55" height="55" /></a></nav>
+  </header>
+  <section class="headline">
+      <h1>Hey there! <br /> I’m Henschel.</h1>
+      <h2>I’am a Software Developer.</h2>
+      <p>I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      <ul class="social-media">
+        <li><a href="https://github.com/miltonHenschel"><img src="images/github.svg" alt="GitHub" width="55" height="55" /></a></li>
+        <li><a href="https://linkedin.com/in/henschelnketchoguem"><img src="images/Linkedin icon.svg" alt="LinkedIn" width="25" height="55" /></a></li>
+        <li><a href="https://www.angellist.com/"><img src="images/angel list.svg" alt="Angellist" width="55" height="55" /></a></li>
+        <li><a href="https://twitter.com/nketchogue"><img src="images/twitter.svg" alt="Twitter" width="55" height="55" /></a></li>
+        <li><a href="https://medium.com/@nketchoguemilton"><img src="images/medium.svg" alt="Medium" width="55" height="55" /></a></li>
+      </ul>
```

Comment:
- [Optional] Consider reducing the size of the social media icons to match the figma template.
### Your design
![1](https://user-images.githubusercontent.com/94551484/198507373-4383312b-c0d2-4944-99d5-e1362fdd3c12.png)

### Figma template
![f1](https://user-images.githubusercontent.com/94551484/198507385-a005ff77-26c8-43b6-88a4-bade01993944.png)

Reply:
Done. Thanks.